modules/ROOT/pages/websockets/websockets-connector-broadcast.adoc

```diff
@@ -0,0 +1,44 @@
+= About Websocket Connection Management Components
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: connectors, ws, wss, websocket, broadcast
+
+== About WebSockets Broadcasting
```

Comment:
This page should be renamed and also include the docs for the send operation which is currently missing